pipeline/07-export.R

```diff
@@ -174,13 +176,72 @@ assessment_pin10_prepped <- assessment_pin_prepped %>%
 for (town in unique(assessment_pin_prepped$township_code)) {
   message("Now processing: ", town_convert(town))
 
-
   ## 4.1. PIN-Level ------------------------------------------------------------
 
+  # Filter building data to specific township
+  assessment_pin10_filtered <- assessment_pin10_prepped %>%
+    filter(township_code == town) %>%
+    select(-township_code)
+
+  building_coords <- assessment_pin10_filtered %>%
+    # Handle a rare edge case where neighborhoods span multiple townships
+    # and so introduce duplicate pin10s; more details here:
+    # https://github.com/ccao-data/data-architecture/issues/275
+    distinct(meta_pin10) %>%
```

Comment:
Do you think it's OK to leave this bandaid in here for now, or do we need to spend some time resolving the root problem?